fetch_gazebo_demo/launch/demo.launch

```diff
@@ -12,9 +12,7 @@
   <node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" >
     <!-- finger tips are 195mm from wrist_roll_link
```

Comment:
these comments should be removed (since the parameters are no longer here) -- consider adding them to the YAML